activerecord/test/cases/persistence_test.rb

```diff
@@ -20,8 +21,68 @@
 require "models/admin"
 require "models/admin/user"
 
+class ReadonlyNameBook < Book
+  attr_readonly :name
+end
+
 class PersistenceTest < ActiveRecord::TestCase
-  fixtures :topics, :companies, :developers, :accounts, :minimalistics, :authors, :author_addresses, :posts, :minivans
+  fixtures :topics, :companies, :developers, :accounts, :minimalistics, :authors, :author_addresses, :posts, :minivans, :books
+
+  def test_insert_many
+    assert_difference "Book.count", +1 do
+      Book.insert_many [{ name: "Rework", author_id: 1 }]
+    end
+  end
```

Comment:
This feature needs more tests since most of them are skipped for adapters other than postgres. Can you add tests that actually insert multiple records (at least 3) and keep this test that shows a single insert works.

Reply:
👍 Sure!

And BTW thank you so much for reviewing my PR! 🙏 — and also for all the work you've been doing on Rails 6!! 👏